README.md

```diff
@@ -83,10 +83,17 @@ Similarly, a `ktlintFormat` task has been added that formats all of the source s
 If project has subprojects - plugin also adds two meta tasks `ktlintCheck` and `ktlintFormat` to the root project that
 triggers related tasks in subprojects.
 
-Another task added: `ktlintApplyToIdea`. It is always added only to the
-root project. Task generates IntelliJ IDEA (or Android Studio) Kotlin
-style files in `.idea/` folder. **Note** that this task will overwrite
-the existing style file.
+### Apply to IDEA
+Two another tasks added:
+- `ktlintApplyToIdea` - Task generates IntelliJ IDEA (or Android Studio) Kotlin
+                        style files in project `.idea/` folder.
+- `ktlintApplyToIdeaGlobally` - Task generates IntelliJ IDEA (or Android Studio) Kotlin
+                                style files in user home IDEA
+                                (or Android Studio) settings folder.
```

Comment:
Home as in `~` or the project root directory?

Reply:
Home is `~` directory. Problem is that it may have different names, like `~/.IntelliIDEA2018`, `~/.IntelliIDEA2017`, `~/.AndroidStudio` or something else.

Reply:
Okay, good. Just wanted clarification. Thanks!